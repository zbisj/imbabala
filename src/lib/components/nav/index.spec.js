import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Nav from ".";

describe("Nav", () => {
  test("renders correctly", () => {
    const tree = renderer
      .create(
        <Nav
          items={[
            { label: "Active" },
            { label: "Link" },
            { label: "Link" },
            { label: "Disabled", link: "https://github.com", disabled: true },
          ]}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
