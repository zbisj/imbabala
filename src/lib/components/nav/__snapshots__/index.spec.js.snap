// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders correctly 1`] = `
<div
  className=""
>
  <ul
    className=""
  >
    <li
      className=""
    >
      <a
        className=""
        href="#"
        onClick={[Function]}
      >
        Active
      </a>
    </li>
    <li
      className=""
    >
      <a
        className=""
        href="#"
        onClick={[Function]}
      >
        Link
      </a>
    </li>
    <li
      className=""
    >
      <a
        className=""
        href="#"
        onClick={[Function]}
      >
        Link
      </a>
    </li>
    <li
      className=""
    >
      <a
        className=""
        disabled={true}
        href="https://github.com"
        onClick={[Function]}
      >
        Disabled
      </a>
    </li>
  </ul>
</div>
`;
